trace4rs/src/handle/loggers.rs

```diff
@@ -134,13 +142,26 @@ impl Layer<SpanBroker> for Logger {
 pub enum EventFormatter {
     Normal,
     MessageOnly,
+    Custom(CustomFormatter),
 }
 
 impl From<ConfigFormat> for EventFormatter {
     fn from(f: ConfigFormat) -> Self {
         match f {
             ConfigFormat::Normal => Self::Normal,
             ConfigFormat::MessageOnly => Self::MessageOnly,
+            ConfigFormat::Custom(s) => {
+                match CustomFormatter::new(s) {
+                    Ok(c) => Self::Custom(c),
+                    #[allow(clippy::print_stderr)] // necessary error surfacing
+                    Err(e) => {
```

Comment:
So, maybe this is pedantic, but would using smartdefault for `EventFormatter` be useful here? Rather than return `Self::Normal` we return `Self::default()`. We still make `Self::Normal` the default, but in this way we establish that it's the defacto default value. Does this make sense?

Reply:
Totally, although for now i've just added a `Default` impl since SmartDefault adds substantially to compile times. I could be convinced otherwise, but thinking we'll delay adding it to our deps until there are a few more use cases.